### PR TITLE
console UI

### PR DESCRIPTION
**Pull request by maen-al-khraisha**
- Create an app.rb file that will serve as a console app entry point. It has methods that do the following:

List all books.
List all people.
Create a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.

- Create main.rb define the entry point, this will be a method called main that is invoked at the end of your file. This method does the following:

Present the user with a list of options to perform.
It Lets users choose an option.
If needed, ask for parameters for the option.
Option to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  end
+
+  def list_people
+    return 'No person added to the library' unless @people.any?
+
+    text = ''
+    @people.each.with_index(1) do |people, i|
+      text += "\n#{i}) Name: \"#{people.name}\" Age: #{people.age}"
+    end
+    text
```
Comment (DeliceLydia) [inline app.rb]: - [x] After creating a `person` that means a person will have his/her personal ID, when checking list of persons you have outputs should be:
` puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"`

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  def list_rental_by_person
+    return 'No person added to the library' unless @rentals.any?
+
+    text = ''
+    @rentals.each.with_index(1) do |rental, i|
+      text += "\n#{i}) Date: \"#{rental.date}\" Person: #{rental.person.name}\" Book: #{rental.book.title}"
+    end
+    text
+  end
```
Comment (DeliceLydia) [inline app.rb]: - [ ] When you have an ID of each person you will be able to List all rentals for a given person's ID, that means before getting a list you should first provide an ID of the person you want to check his/her rentals

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @maenkhraisha, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Awesome README
- Followed git workflow


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app.rb (hunk 1 above)
Comment (maen-al-khraisha) [reply app.rb]: done

**Review by roshan-bajgain — APPROVED** [review]
Hi @maenkhraisha, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**